frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -15,6 +15,14 @@ import { UserInformation } from "./user";
 
 export const APP_BAR_HEIGHT = "64px";
 
+/**
+ * Properties used to allow stories to override featureflag settings
+ */
+interface HeaderProps extends AppConfiguration {
+  showShortLinks?: boolean;
+  showNPS?: boolean;
```

Comment:
can we add some docstrings to these so users know what they mean (specifically NPS since I think it might be unclear what that could stand for)